### PR TITLE
Fix: The issue when the agent has no acronym

### PR DESCRIPTION
**Pull request by Bilelkihal**
Related issue:
https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/636
Capture d’écran 2024-05-31 à 11 58 45

### CODE REVIEW

**Review by Bilelkihal — COMMENTED** [review]
✅

File: app/helpers/agent_helper.rb [hunk 1]
```diff
@@ -169,7 +169,8 @@ def agent_tooltip(agent)
     if agent.affiliations && agent.affiliations != []
       affiliations = ""
       agent.affiliations.each do |affiliation|
-        affiliations = affiliations + affiliation.acronym + " "
```
Comment (syphax-bouazzouni) [inline app/helpers/agent_helper.rb:172]: change to 
```
affiliations = "#{affiliations} #{affiliation.acronym} "
```

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/helpers/agent_helper.rb (hunk 1 above)
Comment (syphax-bouazzouni) [reply app/helpers/agent_helper.rb:172]: and clicking on save does it work?

Comment (Bilelkihal) [reply app/helpers/agent_helper.rb:172]: Yes it works

Comment (syphax-bouazzouni) [reply app/helpers/agent_helper.rb:172]: does the bellow version work ?, if yes use it, you will gain one line.
```
affiliations = "#{affiliations} #{ affiliation.acronym || affiliation.name }"
```

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/helpers/agent_helper.rb (hunk 1 above)
Comment (Bilelkihal) [reply app/helpers/agent_helper.rb:172]: Yes working, thanks

**Review by syphax-bouazzouni — APPROVED** [review]

**Pull request merged**